tests/unit/lib/test_signal_schema.py

```diff
@@ -296,7 +297,7 @@ def test_db_signals_as_columns():
     spec = {"name": str, "age": float, "fr": MyType2}
     lst = list(SignalSchema(spec).db_signals(as_columns=True))
 
-    assert all(isinstance(s, Column) for s in lst)
+    assert all(isinstance(s, sa.Column) for s in lst)
```

Comment:
Any reason why regular `datachain.Column` does not work?

Reply:
It was throwing some strange SQLAlchemy errors when creating table with `datachain.Column`s so this was a workaround (I wanted to fix it in followup as I didn't have time for deeper investigation). Now I did some research and it turns out the problem was that our `datachain.Column` is extending [sa.ColumnClause](https://pydocbrowser.github.io/sqlalchemy/latest/sqlalchemy.sql.elements.ColumnClause.html) which is a parent class of [sa.Column](https://pydocbrowser.github.io/sqlalchemy/latest/sqlalchemy.sql.schema.Column.html#index) which is needed if you want to create a table from it (what I was doing in `DataChain.from_records()` which was calling it). `sa.ColumnClause` cannot be used when defining DB table.

TL;DR I managed to fix it to return `datachain.Column` instead and doing convertion to `sa.Column` in `DataChain.from_records()` which moves these column hacking from user facing method to internal implementation so it's good now I think.